atomicapp/providers/openshift.py

```diff
@@ -449,6 +460,14 @@ def _get_config_values(self):
                            answers[k])
                     logger.error(msg)
                     raise ProviderFailedException(msg)
-        return (result["providerapi"],
-                result["access_token"],
-                result["namespace"])
+
+        logger.debug("config values: %s" % result)
+
+        # set config values
+        self.providerapi = result[PROVIDER_API_KEY]
+        self.access_token = result[ACCESS_TOKEN_KEY]
+        self.namespace = result[NAMESPACE_KEY]
+        if result[PROVIDER_CA_KEY]:
+            self.ssl_verify = result[PROVIDER_CA_KEY]
+        else:
+            self.ssl_verify = result[PROVIDER_TLS_VERIFY_KEY]
```

Comment:
so right now we map `PROVIDER_CA_KEY` and `PROVIDER_TLS_VERIFY_KEY` directly to `ssl_verify`. Will this cause trouble when we merge in the websockets stuff for exec? 

Should we rather leave it and create `self.providerca` and `self.providertlsverify` and then translate that information when we use it. 

So we could then call `make_rest_request(verify=self.do_tls_verify())` and that function would process the information and return back the right information.